### ISSUE TITLE
[CKEditor 5] The new MathType package (v7.18.0) does not work with the latest version of CKEditor 5

### ISSUE THREAD

**Issue by Reinmar** [wiris/html-integrations#16]
The 7.18.0 hardcodes the dependencies to CKEditor 5 packages to specific major versions. Namely, I mean this:

https://github.com/wiris/html-integrations/blob/139dc19edaa6b1db865ce9e5096369445277f73d/packages/mathtype-ckeditor5/package.json#L38-L41

This means that MathType cannot be used with CKEditor 5 v17.0.0. What people will see is the "some modules are duplicated" error.

What you should be doing is specifying the lowest known version of those CKEditor 5 dependencies with which MathType is known to work.

```plaintext
    "@ckeditor/ckeditor5-core": ">=16.0.0",
    "@ckeditor/ckeditor5-engine": ">=16.0.0",
    "@ckeditor/ckeditor5-ui": ">=16.0.0",
    "@ckeditor/ckeditor5-widget": ">=16.0.0",
```

This is a critical issue as it makes the plugin unusable.

**Comment by Naterra** [comment]
+1

**Comment by xaviripo** [comment]
Fixed in 50f4098eda4d871d12e300b3e85ff0e8b5db310f.
